.ci/jenkins/pipeline/proj-jjb.yaml

```diff
@@ -140,17 +138,95 @@
                     parent-credentials: true
         script-path: "{jjb_jenkinsfile}"  # Path to Jenkinsfile that defines the build steps
 
+# Template for NIXLBench container build job
+# Builds and pushes NIXLBench container images for x86_64 and aarch64
+# Supports nightly automatic builds and manual builds via parameters
+- job-template:
+    name: "{jjb_proj}-nixlbench-container-build"
+    project-type: pipeline
+    disabled: false
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+          num-to-keep: 30
+      - inject:
+          keep-system-variables: true
+          properties-content: |
+            jjb_proj={jjb_proj}-nixlbench-container-build
+            conf_file=.ci/jenkins/lib/nixlbench-container-build-matrix.yaml
+    description: >
+      <b>NIXLBench container build</b><br/>
+      • Builds and pushes x86_64 &amp; aarch64 images with any NIXL/UCX version combination<br/>
+      • Optional latest tag update via <code>UPDATE_LATEST</code> parameter<br/>
+      • All images pushed to unified path: <code>verification/nixlbench/</code><br/>
+      <br/>
+      <i>Do NOT edit this job through the Jenkins GUI &mdash; managed by Jenkins Job Builder.</i>
+    concurrent: false
+    sandbox: true
+    # Nightly scheduler for automatic builds with default versions
+    triggers:
+      - timed: "H 3 * * *" # Run nightly around 3 AM (H adds randomness within the hour)
+    # Manual build parameters
+    parameters:
+      - string:
+          name: "NIXL_VERSION"
+          default: "{jjb_branch}"
+          description: "NIXL version to use (tag like 0.3.1, branch name, or commit hash)"
+      - string:
+          name: "UCX_VERSION"
+          default: "master"
```

Comment:
v1.19.x

Reply:
[CI: Change the default UCX ver](https://github.com/ai-dynamo/nixl/pull/436/commits/ecca82cf3e5d0223c4e381b2915de0dc2f8cffe6)